x-pack/legacy/plugins/apm/server/lib/helpers/round_nice.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function roundNice(v: number) {
```

Comment:
I feel like this file could use a more descriptive name (maybe `roundToNearestFiveOrTen`?) and `v` could be something like `value`. Maybe a comment w/ some examples as well?

Reply:
Good point. I did add a test which I feel is fairly descriptive but adding examples as comments directly next to the implementation will be an improvement.